site/cypress/integration/autograding_status_1.spec.js

```diff
@@ -0,0 +1,97 @@
+            cy.get('#autograding-status-table tbody tr td').eq(6).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(7).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(8).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(9).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(10).then(element => cy.get(element).should('contain', '101'));
+
+            // add a non regrade job and see that the site is updated
+
+            cy.visit(`/courses/${getCurrentSemester()}/sample/gradeable/future_no_tas_homework`);
+            cy.wait(500);
+            cy.get('.upload-box').attachFile('sample_upload.py', { subjectType: 'drag-n-drop' });
+            cy.get('#submit').click();
+            cy.wait(500);
+
+            cy.visit(autograding_status_path);
+
+            cy.get('#toggle-btn').should('have.text', 'Pause Update').click().should('have.text', 'Resume Update');
+            cy.get('#course-table tbody tr td').eq(0).then(element => cy.get(element).should('contain', getCurrentSemester()));
+            cy.get('#course-table tbody tr td').eq(1).then(element => cy.get(element).should('contain', 'sample'));
+            cy.get('#course-table tbody tr td').should('contain', 'closed_homework');
+            cy.get('#course-table tbody tr td').should('contain', 'future_no_tas_homework');
+            cy.get('#course-table tbody tr td').should('contain', '1');
+
+            cy.get('#autograding-status-table tbody tr td').eq(1).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(2).then(element => cy.get(element).should('contain', '1'));
+            cy.get('#autograding-status-table tbody tr td').eq(3).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(4).then(element => cy.get(element).should('contain', '101'));
+            cy.get('#autograding-status-table tbody tr td').eq(5).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(6).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(7).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(8).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(9).then(element => cy.get(element).should('contain', '0'));
+            cy.get('#autograding-status-table tbody tr td').eq(10).then(element => cy.get(element).should('contain', '102'));
+
+            cy.logout();
```

Comment:
You might think about making a function which runs after each test and logs out the user instead of having to do it in each test.